cranelift/codegen/src/isa/pulley_shared/inst/regs.rs

```diff
@@ -89,10 +89,10 @@ define_registers! {
     x_reg(24) => x24, writable_x24;
     x_reg(25) => x25, writable_x25;
     x_reg(26) => x26, writable_x26;
+    x_reg(27) => x27, writable_x27;
+    x_reg(28) => x28, writable_x28;
 
-    x_reg(27) => stack_reg, writable_stack_reg;
-    x_reg(28) => lr_reg, writable_lr_reg;
-    x_reg(29) => fp_reg, writable_fp_reg;
+    x_reg(29) => stack_reg, writable_stack_reg;
     x_reg(30) => spilltmp_reg, writable_spilltmp_reg;
     x_reg(31) => spilltmp2_reg, writable_spilltmp2_reg;
```

Comment:
Do we actually use spilltmp2 in pulley? AFAICT, we don't. Could free up one more GPR I think.